app/service/contact_svc.py

```diff
@@ -115,6 +116,14 @@ def __init__(self, agent_config):
         except Exception:
             pass
 
+    async def load_c2_config(self, directory):
+        c2_configs = {}
```

Comment:
hm-- this function is currently unused.

Reply:
in stockpile

Reply:
This is used in dark for DNS C2.

Reply:
ah-- it shouldn't :) 

we were using YML files to describe the C2 options. over the weekend I dropped that requirement.